tests/evaluate/test_evaluation.py

```diff
@@ -1679,7 +1679,13 @@ def test_binary_classification_missing_minority_class_exception_override(
         )
     _, saved_metrics, _, _ = get_run_data(run.info.run_id)
 
-    assert saved_metrics == eval_result.metrics
+    for key, saved_val in saved_metrics.items():
```

Comment:
this test can also be fixed by removing line 1670 above, but it seems intentional so i left it